CrazyCanvas/Include/GUI/HUDGUI.h

```diff
@@ -39,15 +41,15 @@ class HUDGUI : public Noesis::Grid
 	HUDGUI(const LambdaEngine::String& xamlFile);
 	~HUDGUI();
 
+	bool ConnectEvent(Noesis::BaseComponent* pSource, const char* pEvent, const char* pHandler) override;
+
 	void OnButtonGrowClick(Noesis::BaseComponent* pSender, const Noesis::RoutedEventArgs& args);
 	void OnButtonShootClick(Noesis::BaseComponent* pSender, const Noesis::RoutedEventArgs& args);
 	void OnButtonScoreClick(Noesis::BaseComponent* pSender, const Noesis::RoutedEventArgs& args);
 
-	bool ConnectEvent(Noesis::BaseComponent* pSource, const char* pEvent, const char* pHandler) override;
-
 	bool ApplyDamage(float damage);
 	bool UpdateScore();
-	bool UpdateAmmo();
+	bool UpdateAmmo(const int32 currentAmmo, const int32 ammoCap);
```

Comment:
I like declaring things as const too, but not for parameters that are copied by value.

Reply:
Alright